R/prepare_data_waterfall.R

```diff
@@ -18,11 +18,12 @@ prepare_data_waterfall <- function(data,
   }
 
   region_filter_2 <- "Latin America and the Caribbean"
+  # TODO: Remove dead code
   company_type_filter <- 1
 
   data_portfolio <- data %>%
     filter(.data$dual == 1) %>% # for now we only look at not dual plants as we don't know how to deal with dual - Angelika
-    # TODO: Ask Souad if this is dead code
+    # TODO: Remove dead code?
     #    filter(.data$CA100 == company_type_filter) %>%
     #    filter(.data$SBTI == company_type_filter) %>%
     #    filter(.data$comp_region == region_filter_2) %>%
```

Comment:
@Souad-2DII, while we're here -- can we delete the code that is commented out? There is more in the same file.